services/data_plane_manager/src/main/java/com/futurewei/alcor/dataplane/cache/LocalCacheImpl.java

```diff
@@ -127,11 +150,12 @@ public void updateLocalCache(NetworkConfiguration networkConfig) throws Exceptio
         OperationType opType = networkConfig.getOpType();
         switch (opType) {
             case CREATE:
-                addSubnetPorts(networkConfig);
-                break;
             case UPDATE:
-                updateSubnetPorts(networkConfig);
+                addSubnetPorts(networkConfig);
```

Comment:
If CREATE and UPDATE has the same implementation, you are actually "setting" a port. Maybe rename the method to setSubnetPorts

Reply:
`addSubnetPorts` has been changed to `setSubnetPorts`